### PR TITLE
[Issue 221] Open source group kick deliverable specification draft

### LINKED ISSUE #221: [Deliverable Spec]: Open source onboarding process
### Description

Create the individual milestone document for the Open source onboarding process 30k #72 

### Definition of Done

- [x] deliverable spec created in the public wiki

### PR DESCRIPTION
**Pull request by sumiat**
first draft of the deliverable specification document

## Summary
Fixes #221

### CODE REVIEW

File: documentation/deliverables/individual_deliverables/open-source-group-kickoff.md
Comment (lucasmbrown-usds) [inline documentation/deliverables/individual_deliverables/open-source-group-kickoff.md:12]: Nit: for some reason the rendered formatting here for the bullets isn't working. Maybe need a space between `-` and `**`

File: documentation/deliverables/individual_deliverables/open-source-group-kickoff.md
Comment (lucasmbrown-usds) [inline documentation/deliverables/individual_deliverables/open-source-group-kickoff.md:15]: Need more indents here?

File: documentation/deliverables/individual_deliverables/open-source-group-kickoff.md
Comment (lucasmbrown-usds) [inline documentation/deliverables/individual_deliverables/open-source-group-kickoff.md:31]: Is this bullet more appropriate for the PAC Kickoff 30k?

**Review by lucasmbrown-usds — COMMENTED** [review]
I understand we're downscoping this quite a bit, so I'm very open to having a conversation around what we want to keep in and move out of the definition of done. 

This to me feels like we would want to rename it as something like, "Communication tools 2.0" or something like that. Or "Open source comms tools"?

With the original proposal for Open Source Group kickoff, I think we were hoping to have the first of an ongoing set of public, open source meetings on the project, setup a mailing list for those participants, etc. I'm okay if we defer that to another 30k, but we should probably update the name of the 30k for clarity.

**Comment by sumiat** [comment]
> With the original proposal for Open Source Group kickoff, I think we were hoping to have the first of an ongoing set of public, open source meetings on the project, setup a mailing list for those participants, etc. I'm okay if we defer that to another 30k, but we should probably update the name of the 30k for clarity.

That makes sense @lucasmbrown-usds! We can definitely rename it, and I would love to understand what original expectations were. We may be able to set up a public calendar, and/or set up the first meeting(s) or welcome session. I was more conservative on what we could get done because I wanted to wait until the Open Source Evangelist had started.

File: documentation/deliverables/individual_deliverables/open-source-group-kickoff.md
Comment (sumiat) [reply documentation/deliverables/individual_deliverables/open-source-group-kickoff.md:31]: I can be more clear here. I was thinking if we wanted to have an open source group within the PAC, but maybe that's not needed or a priority for this.

File: documentation/deliverables/individual_deliverables/open-source-group-kickoff.md
Comment (widal001) [inline documentation/deliverables/individual_deliverables/open-source-group-kickoff.md:7]: Can you update this to match the [summary table in the other deliverable specs](https://github.com/HHS/simpler-grants-gov/blob/main/documentation/deliverables/individual_deliverables/static_site_public_launch_infrastructure_content.md#static-site-public-launch-infrastructure-and-content)?

File: documentation/deliverables/individual_deliverables/open-source-group-kickoff.md
Comment (widal001) [inline documentation/deliverables/individual_deliverables/open-source-group-kickoff.md:1]:
```suggestion
# Open Source Onboarding Process
```
Since we're mainly focusing on implementing the comms tools for open source group and setting up the process to onboard new open source participants, I wonder if this is a more accurate title?

File: documentation/deliverables/individual_deliverables/open-source-group-kickoff.md
Comment (widal001) [reply documentation/deliverables/individual_deliverables/open-source-group-kickoff.md:31]: Agree that this seems a bit outside the scope of this 30k as it's currently defined, but a good goal to save for [that 30k](https://github.com/HHS/simpler-grants-gov/issues/327)

File: documentation/deliverables/individual_deliverables/open-source-group-kickoff.md
Comment (widal001) [inline documentation/deliverables/individual_deliverables/open-source-group-kickoff.md:31]:
```suggestion
- Set up the foundational set of communication channels for continuous user input, feedback, and engagement for an open source project
- Ensure that public code contributions meet code quality and security standards
- Build stakeholder trust in our product roadmap and our approach to development
- Streamline and standardize the onboarding process for new open source contributors
```
Could be good to highlight that an important goal of this 30k is to standardize and increase the efficiency of our onboarding process

File: documentation/deliverables/individual_deliverables/open-source-group-kickoff.md
Comment (widal001) [inline documentation/deliverables/individual_deliverables/open-source-group-kickoff.md:36]: Love this! Good callout

File: documentation/deliverables/individual_deliverables/open-source-group-kickoff.md
Comment (widal001) [inline documentation/deliverables/individual_deliverables/open-source-group-kickoff.md:39]: Yes! Love this too!

File: documentation/deliverables/individual_deliverables/open-source-group-kickoff.md
Comment (widal001) [inline documentation/deliverables/individual_deliverables/open-source-group-kickoff.md:38]:
```suggestion
  -  have streamlined tools and processes for managing contributions, reviewing code, and ensuring code quality, so I can efficiently maintain and enhance the project while upholding its standards.
  - to have a standardized process for onboarding new open source contributors, so that members of the public can be added to our communication channels smoothly and efficiently.
```

File: documentation/deliverables/individual_deliverables/open-source-group-kickoff.md
Comment (widal001) [inline documentation/deliverables/individual_deliverables/open-source-group-kickoff.md:62]:
```suggestion
```
Just removing some of the extra spaces

File: documentation/deliverables/individual_deliverables/open-source-group-kickoff.md
Comment (widal001) [inline documentation/deliverables/individual_deliverables/open-source-group-kickoff.md:110]: @lucasmbrown-usds Do we anticipate creating translations of the content in the public wiki?

I would say actually translating the content is out of scope for this 30k, but we should make a note of it if we expect the public wiki and the onboarding guides to be translated in the future.

File: documentation/deliverables/individual_deliverables/open-source-group-kickoff.md
Comment (widal001) [inline documentation/deliverables/individual_deliverables/open-source-group-kickoff.md:119]:
```suggestion
- Zoom for video conferencing. Only internal teams will have a Zoom license, but the public will join Zoom
- **Google group:** We'll use this for email-based communication with our open source community.
- **Public wiki:** We'll be using this to share public information about the project, such as onboarding guides, presentations, and meeting notes for public meetings.
```

File: documentation/deliverables/individual_deliverables/open-source-group-kickoff.md
Comment (widal001) [inline documentation/deliverables/individual_deliverables/open-source-group-kickoff.md:132]: Not sure if this is still a goal?

File: documentation/deliverables/individual_deliverables/open-source-group-kickoff.md
Comment (widal001) [inline documentation/deliverables/individual_deliverables/open-source-group-kickoff.md:134]: Do we have an ADR for document sharing?

File: documentation/deliverables/individual_deliverables/open-source-group-kickoff.md
Comment (widal001) [inline documentation/deliverables/individual_deliverables/open-source-group-kickoff.md:141]:
```suggestion
- No, this 30k deliverable does not involve sharing any new production data.
```

File: documentation/deliverables/individual_deliverables/open-source-group-kickoff.md
Comment (widal001) [inline documentation/deliverables/individual_deliverables/open-source-group-kickoff.md:147]: Let's not forget to fill this out! The main security consideration that stands out to me is:

- Adding members of the public to slack, which contains a mix of public and private channels

And the mitigation strategy I'd highlight is:

- We have policy guidelines around where to post sensitive content.
- We've trained internal staff on those policies
- We'll review public channels for sensitive content and remove that content prior to inviting open source contributors.

File: documentation/deliverables/individual_deliverables/open-source-group-kickoff.md
Comment (widal001) [inline documentation/deliverables/individual_deliverables/open-source-group-kickoff.md:97]: I would say yes, though we can have a very minimal first version of the public wiki.

File: documentation/deliverables/individual_deliverables/open-source-group-kickoff.md
Comment (widal001) [inline documentation/deliverables/individual_deliverables/open-source-group-kickoff.md:92]:
```suggestion
- **Participant advisory council (PAC):** We will not have a participant advisory council in place when work begins on this deliverable. Instead, this deliverable will be required to onboard members of the PAC once we begin work on setting it up.
- **Site content translations:** We will not yet have a process in place to translate the content of our static site or repository documents. That will be addressed in a future 30k ft deliverable.
```

**Review by widal001 — CHANGES_REQUESTED** [review]
I really like how you narrowed the focus of this 30k deliverable spec to focus specifically on finalizing comms tool implementation and onboarding open source community members!

I left a bunch of suggestions throughout, but I think the major ones are:

- As we talked about last week, we should rename this deliverable. Maybe "Open source onboarding process" or "Open source group onboarding"?
- We should add public wiki and google group to the scope of this 30k
- We should set some explicit targets in the DoD for the minimum number of folks onboarded to each of our public-facing tools to call this "done" -- that's the best test for whether or not we have a repeatable process.
- Also, there are a couple of sections of the template that still need to be filled out or expanded, namely:
  - Translations
  - Security considerations
  - Not doing - I just thought it could be good to explicitly state that we won't be holding the first open source group meeting.

File: documentation/deliverables/individual_deliverables/open-source-group-kickoff.md
Comment (sumiat) [reply documentation/deliverables/individual_deliverables/open-source-group-kickoff.md:7]: done! Billy, I wasn't sure if I should mark you as a product owner or as another document owner since you would be contributing to this document too. Feel free to change it.

File: documentation/deliverables/individual_deliverables/open-source-group-kickoff.md
Comment (sumiat) [reply documentation/deliverables/individual_deliverables/open-source-group-kickoff.md:75]: I will include this in the security section but I believe we will need to get security approval to use Google Groups and maybe Zoom since they will be collecting email addresses of users. Slack and wiki are used currently so I assume they are OK but we may need to check on that too.

Comment (sumiat) [reply documentation/deliverables/individual_deliverables/open-source-group-kickoff.md:75]: Added some of your comments into the spec!

File: documentation/deliverables/individual_deliverables/open-source-group-kickoff.md
Comment (sumiat) [reply documentation/deliverables/individual_deliverables/open-source-group-kickoff.md:104]: I will include this in the security section but I believe we will need to get security approval to use Google Groups and maybe Zoom since they will be collecting email addresses of users. Slack and wiki are used currently so I assume they are OK but we may need to check on that with the Security team too

File: documentation/deliverables/individual_deliverables/open-source-group-kickoff.md
Comment (sumiat) [reply documentation/deliverables/individual_deliverables/open-source-group-kickoff.md:136]: We do. Here: https://github.com/HHS/simpler-grants-gov/issues/118

File: documentation/deliverables/individual_deliverables/open-source-group-kickoff.md
Comment (widal001) [reply documentation/deliverables/individual_deliverables/open-source-group-kickoff.md]: This isn't as relevant if we'll be moving these into the public wiki!

**Comment by sumiat** [comment]
Closing this PR as I moved the deliverable spec to the public wiki and will publish from there.

**Pull request closed without merging**
